cmd/common/utils/errors.go

```diff
@@ -112,9 +112,9 @@ func WrapInErrInvalidArg(arg string, err error) error {
 // JSON parsing
 
 func WrapInErrUnmarshalOutput(err error, output string) error {
-	return fmt.Errorf("failed to unmarshal output: %w\n%s", err, output)
+	return fmt.Errorf("unmarshaling output: %w\n%s", err, output)
 }
 
 func WrapInErrMarshalOutput(err error) error {
-	return fmt.Errorf("failed to marshal output: %w", err)
+	return fmt.Errorf("marshalling output: %w", err)
```

Comment:
```suggestion
	return fmt.Errorf("marshaling output: %w", err)
```
Both are valid (American vs British English). I suggest to use a single `l` to be consistent with other errors